src/commands/functions/functions-create.js

```diff
@@ -189,6 +198,42 @@ const pickTemplate = async function ({ language: languageFromFlag }) {
 
 const DEFAULT_PRIORITY = 999
 
+const selectTypeOfFunc = async () => {
+  const { functionType } = await inquirer.prompt([
+    {
+      name: 'functionType',
+      message: "Select the type of function you'd like to create: ",
```

Comment:
I think we can leave this type of formatting to the prompt library.

```suggestion
      message: "Select the type of function you'd like to create",
```